src/components/common/DocumentDelivery.vue

```diff
@@ -58,6 +67,13 @@ export default class DocumentDelivery extends Vue {
   @Prop({ default: false }) readonly editableCompletingParty: boolean
   @Prop({ default: false }) readonly showCustodianEmail: boolean
   @Prop({ default: false }) readonly invalidSection: boolean
+  // additionaLabel can be used for additon fields.
+  //  For SP can be used for proprietor
+  // FOr GP label can be Partners
+  @Prop({ default: null }) readonly additionalLabel: string
```

Comment:
1. I think you don't need these comments: Developers will figure out what this can be used for, and now you've made this component filing-specific. Remove them :)

2. If you do need to add comments, use JDoc style so that developers can hover over the method/prop and see the comment as contextual help.